polkadot/node/subsystem-bench/src/lib/configuration.rs

```diff
@@ -35,24 +35,39 @@ pub struct PeerLatency {
 	pub std_dev: f64,
 }
 
-// Default PoV size in KiB.
+// Based on Kusama `max_validators`
+fn default_n_validators() -> usize {
+	300
+}
+
+// Based on number of kusama validators producing blocks
+fn default_n_cores() -> usize {
+	50
+}
+
+// PoV size, KiB. Based on empirical data from Kusama
 fn default_pov_size() -> usize {
-	5120
+	2 * 1024
 }
 
-// Default bandwidth in bytes
+// Default bandwidth in bytes, based stats from Kusama validators
 fn default_bandwidth() -> usize {
-	52428800
+	42 * 1024 * 1024
+}
+
+// Default peer latency
+fn default_peer_latency() -> Option<PeerLatency> {
+	Some(PeerLatency { mean_latency_ms: 30, std_dev: 2.0 })
 }
 
 // Default connectivity percentage
 fn default_connectivity() -> usize {
-	100
+	90
 }
 
-// Default backing group size
+// Based on Kusama `max_validators_per_core`
 fn default_backing_group_size() -> usize {
-	5
+	3
```

Comment:
This has to be kept in sync with the actual value from chain. Let's keep it 5 here and override in specific benchmarks.